module/modmanager/manager.go

```diff
@@ -1245,8 +1245,13 @@ func (m *module) registerResources(mgr modmaninterface.ModuleManager, logger log
 							m.logger.Errorf("error in modular DiscoverComponents: %s", err)
 							return nil, err
 						}
-
-						return res, nil
+						if len(res.Discovery) > 1 {
+							return nil, errors.New("modular DiscoverComponents response contains more than one discovery")
```

Comment:
I think errors in golang are supposed to start lower cased?

Reply:
it does

Reply:
this is lowercase tho